[BOP-947] Known issue: Managed user passwords not migrated from MKE 3… (#55)
Mirantis/mke-docs: content/docs/release-notes/known-issues.md

   ```sh
   mkectl apply
   ```

## [BOP-947] Managed user passwords are not migrated during upgrade from MKE 3

> Admin passwords are migrated during upgrade from MKE 3, however managed user

Reviewer: ```suggestion
`admin` password is migrated during upgrade from MKE 3, however all other managed user
```
